vhdbuilder/scripts/linux/tool_installs.sh

```diff
@@ -45,4 +45,28 @@ disableSystemdIptables() {
 
 enableCgroupV2forAzureLinux() {
     sed -i 's/systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0//g' /boot/systemd.cfg
+}
+
+# download and setup azcopy to use to download private packages with MSI auth
+getAzCopyCurrentPath() {
+  if [[ -f ./azcopy ]]; then
+    echo "./azcopy already exists"
+  else
+    echo "get azcopy at \"${PWD}\"...start"
+    # Download and extract
+    azcopydownloadurl="https://aka.ms/downloadazcopy-v10-linux"
```

Comment:
This binaries are not being published with checksum, it might be a bit risky to just download from a redirected url and use (even it's from aka.ms) as it opens risks on supply chain attack. I would suggest we download from the canonical urls for now and compare with pre-calculated sha256 checksums.

I also sent a follow up comment in azcopy's issue: https://github.com/Azure/azure-storage-azcopy/issues/2411

Reply:
updated per suggestion, thank you!

Reply:
@AbelHu Abel, this will apply to the azcopy installation for windows as well, i haven't updated that code, just fyi